Disable unmanaged topics reconciliation (#10481)
strimzi/strimzi-kafka-operator: documentation/modules/operators/proc-converting-managed-topics.adoc

+
[source,shell,subs="+quotes"]
----
kubectl get kafkatopic my-topic-1 -o yaml
----
+

> .Example topic with the `Unmanaged` status

Reviewer: ```suggestion
.Example topic with an `Unmanaged` status
```

  generation: 124
  name: my-topic-1
  finalizer: 
  - strimzi.io/topic-operator
  labels:
    strimzi.io/cluster: my-cluster
  annotations:
    strimzi.io/managed: "false"
spec:
  partitions: 10
  replicas: 2
  config:
    retention.ms: 432000000
status: 
  observedGeneration: 124 # <1>
  conditions:
  - lastTransitionTime: "2024-08-22T06:07:57.671085635Z"
    status: "True"
    type: Unmanaged # <2>
----
<1> The value of `metadata.generation` must `match status.observedGeneration`.

> <2> The Unmanaged condition means that the KafkaTopic is not reconciled anymore.

Reviewer: ```suggestion
<2> The `unmanaged` condition means that the `KafkaTopic` is no longer reconciled.
```
